src/main/java/io/debezium/connector/yugabytedb/YugabyteDBChangeEventSourceCoordinator.java

```diff
@@ -155,6 +155,10 @@ protected void executeChangeEventSources(CdcSourceTaskContext taskContext,
 
     @Override
     public void commitOffset(Map<String, ?> offset) {
+        if (snapshotSource == null) {
```

Comment:
if not local variable use `this`. Should follow that convention everywhere.
That way it becomes easy to see if null check is required or not.